tools/release/prepare-release.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+"""
+Use this script right before a release to:
+- Add a new entry to the version-selection menu in the documentation.
+- Update headings in release notes and add new empty sections for next release.
+
+BEFORE running this script update the list of versions and targets to include the
+new desired version (or remove old unused versions).
+"""
+
+from datetime import datetime
+import pathlib
+
+
+def make_version_select(root):
+    version_select = f'{root}/docs/_templates/topbar/launchbuttons.html'
+
+    with open(f'{root}/tools/release/templates/launchbuttons.html', 'r') as f:
+        contents = f.readlines()
+
+    versions = ['0.11', '0.10', '0.9', '0.8', '0.7', '0.6', '0.5', '0.4', '0.3']
+    targets = [
+        '0.11.0', '0.10.1', '0.9.0', '0.8.4', '0.7.1', '0.6.1', '0.5.0', '0.4.0',
+        '0.3.0'
+    ]
+
+    entries = []
+    for version, target in zip(versions, targets):
+        entries.append(f"""        <a class="dropdown-buttons"
+            href="https://scipp.github.io/release/{target}"><button type="button"
+                class="btn btn-secondary topbarbtn">v{version}</button></a>\n""")
+
+    contents.insert(14, ''.join(entries))
+
+    with open(version_select, "w") as f:
+        contents = "".join(contents)
+        f.write(contents)
+
+
+def update_release_notes(root):
```

Comment:
Yes it will, but I don't like trickery either (especially if it can break equally easily). Let us change it when it breaks. Since this does not commit anything the user of this script should be able to tell from the diff.